platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -122,10 +122,10 @@ See the [Compatibility Matrix](userguide/compatibility.html) for more informatio
 <a name="kotlin-2"></a>
 ### Update to Kotlin 2
 
-Gradle embeds the latest stable release of [Kotlin 2.1.x](userguide/compatibility.html#kotlin) runtime and uses [Kotlin language version 2.1](userguide/compatibility.html#kotlin).
+Gradle embeds the latest stable release of [Kotlin 2.2.x](userguide/compatibility.html#kotlin) runtime and uses [Kotlin language version 2.2](userguide/compatibility.html#kotlin).
 This marks a shift from Gradle 8.x, which embedded Kotlin 2.0 starting in 8.11 but continued to use Kotlin language version 1.8 for compatibility.
 
-For a comprehensive overview of what’s new, see the [Kotlin 2.1.0](https://kotlinlang.org/docs/whatsnew21.html) and [Kotlin 2.0.0](https://kotlinlang.org/docs/whatsnew20.html) release notes for full details.
+For a comprehensive overview of what’s new, see the [Kotlin 2.2.0](https://kotlinlang.org/docs/whatsnew22.html), [Kotlin 2.1.0](https://kotlinlang.org/docs/whatsnew21.html) and [Kotlin 2.0.0](https://kotlinlang.org/docs/whatsnew20.html) release notes for full details.
```

Comment:
It's not directly related to this PR but I just noticed the start of this sentence says "for a comprehensive overview" then lists links, then after the links it says "for full details". That feels redundant and could be updated:

```suggestion
For a comprehensive overview of what’s new, see the [Kotlin 2.2.0](https://kotlinlang.org/docs/whatsnew22.html), [Kotlin 2.1.0](https://kotlinlang.org/docs/whatsnew21.html) and [Kotlin 2.0.0](https://kotlinlang.org/docs/whatsnew20.html) release notes.
```